src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] <b>REQUIRED:</b> Project requires that you import `App` component here not `Calculator`.  `Calculator` component must be imported in the App component instead. Kindly do that to comply with project requerements.

Reply:
Thank you for pointing this out, I have effected the changes as recommended